### PR TITLE
[Feature] Food Carts

### PR DESCRIPTION
**Pull request by FalseIncarnate**
Adds brand new Food Carts!
- Can carry 6 food items and 6 drink items
- One spawns in the kitchen, additional ones may be ordered from cargo
for 10 supply points each
- Sprite lifted from TG as per requests by multiple people. Old sprite is still present in code for possible use in the future.

image

Removes the OPENCONTAINER flag and transfer amount from the engicart.
- This resolves a runtime only Cole would encounter, but still won't let
him put vodka in the engicart.

All carts (jani, engi, and food) can be wrenched to lock them into place.
- This should avoid the janitor getting his cart pushed halfway down the hall while mopping.

### CODE REVIEW

File: paradise.dme [hunk 1]
```diff
@@ -1340,7 +1341,7 @@
 #include "code\modules\mob\living\simple_animal\friendly\corgi.dm"
 #include "code\modules\mob\living\simple_animal\friendly\crab.dm"
 #include "code\modules\mob\living\simple_animal\friendly\farm_animals.dm"
-#include "code\modules\mob\living\simple_animal\friendly\fox.dm"
```
Comment (Markolie) [inline paradise.dme]: Looks good, aside from this.

**Comment by Markolie** [comment]
Also, rather than listing food1, food2, drink1, drink2, etc. It might be better to have two lists, food and drinks, and then looping through those with a for loop and checking for their index. That way you can compress all those separate if checks down to three or so loops.

**Comment by FalseIncarnate** [comment]
Fixed the issue with Fox and fox in the .dme that I missed.

**Comment by Fox-McCloud** [comment]
What's up with replacing the entirety of the cargo supply datum?

In any event, don't like the sprite, at all; the one that FJ posted on the forums (I believe it was there anyway) was much better.

**Comment by fj45** [comment]
https://camo.githubusercontent.com/f367b33b47b2b26b5197f86d114de0dd4dc9b30b/68747470733a2f2f692e696d6775722e636f6d2f61384d4e4f6d442e706e67

This one.

**Comment by Fox-McCloud** [comment]
Yeah, that one; that one is way better, in my opinion and looks more like a foodcart.

**Comment by FalseIncarnate** [comment]
Swapped sprites and made some additional changes. Functionality remains relatively unchanged.

**Pull request merged**